.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*]
+indent_size = 2
+indent_style = tab
+
+[*.json]
+indent_size = 2
```

Comment:
[EOF](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)
https://thoughtbot.com/blog/no-newline-at-end-of-file